### PR TITLE
add group_vars

### PR DESCRIPTION
**Pull request by Mityushin**
refactor playbooks
add new hierarchy
playbooks
| - examples
| - simple
| - tests

### CODE REVIEW

**Review by yurkov-anton — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,30 +2,46 @@
...
+| cifsShare        |    +    |    -    |    -    |    -    |
+| fileInterface    |    +    |    +    |    +    |    -    |
+| host             |    +    |    +    |    +    |    -    |
+| iscsiPortal      |    +    |    +    |    +    |    -    |
+| lun (reference to storageResource)  |    +    |    +    |    +    |    -    |
+| nasServer        |    +    |    +    |    +    |    -    |
+| nfsServer        |    +    |    +    |    +    |    -    |
+| pool             |    +    |    +    |    +    |    -    |
+| storageResource  |         |         |         |    -    |
+
```
Comment (marsofandrew) [inline README.md:23]: They are all supported action (option) get

File: README.md [hunk 2]
```diff
@@ -34,27 +50,28 @@ The task's other options does the following in the order presented below:
 2. Update the default password of the Unity system admin user;
 3. Upload license to Unity.
 
-For updating and querying the Unity system, please refer to the sample task in test-1.yml:
+For updating and querying the Unity system, please refer to the sample task in <ВСТАВЬ>.yml:
 
+    - name: НАПИСАТЬ ПЛЕЙБУК
```
Comment (marsofandrew) [inline README.md:55]: what is this?

File: README.md [hunk 3]
```diff
@@ -34,27 +50,28 @@ The task's other options does the following in the order presented below:
...
+        - {resource_type: dnsServer, id: '0', addresses: [10.254.66.23, 10.254.66.24]}
+        - {resource_type: ntpServer, id: '0', addresses: [10.254.140.21, 10.254.140.22], rebootPrivilege: 2}
+        # - {resource_type: user, id: 'user_test1', action: 'delete'}
+      unity_password_updates:
+        # - {username: test1, password: Welcome1!, new_password: Welcome2!}
+      unity_queries:
+        # - {resource_type: user, id: 'user_test1', fields: 'role.id'}
+        - {resource_type: remoteSyslog, id: "0", fields: 'address,protocol,facility,enabled'}      # id parameter has to be of the string type
+        - {resource_type: dnsServer, fields: "domain, addresses, origin", page: 1, per_page: 100}
+        - {resource_type: ntpServer, id: "0", fields: addresses}      # id parameter has to be of the string type
```
Comment (marsofandrew) [inline README.md:74]: it's old version, and now it doesn't work

File: README.md [hunk 4]
```diff
@@ -72,6 +89,7 @@ Password updates are distinct from other updates, because you can neither query
 
 To delete a resource instance, you use an update option with the resource type, instance ID, and the action 'delete'. Here is the sample task in test-2.yml:
 
+    - name: НАПИСАТЬ ПЛЕЙБУК
```
Comment (marsofandrew) [inline README.md:92]: Write "TODO: " if you want

File: README.md (hunk 2 above)
Comment (Mityushin) [reply README.md:55]: TODO:

File: README.md (hunk 3 above)
Comment (Mityushin) [reply README.md:74]: Yes, this will be rewrite when all ours version will be merged.

**Review by marsofandrew — APPROVED** [review]

**Pull request merged**